test/test_datasets_video_utils.py

```diff
@@ -40,46 +41,46 @@ def test_unfold(self):
             [0, 1, 2],
             [3, 4, 5],
         ])
-        self.assertTrue(r.equal(expected))
+        assert_equal(r, expected, check_stride=False)
 
         r = unfold(a, 3, 2, 1)
         expected = torch.tensor([
             [0, 1, 2],
             [2, 3, 4],
             [4, 5, 6]
         ])
-        self.assertTrue(r.equal(expected))
+        assert_equal(r, expected, check_stride=False)
 
         r = unfold(a, 3, 2, 2)
         expected = torch.tensor([
             [0, 2, 4],
             [2, 4, 6],
         ])
-        self.assertTrue(r.equal(expected))
+        assert_equal(r, expected, check_stride=False)
 
     @unittest.skipIf(not io.video._av_available(), "this test requires av")
     def test_video_clips(self):
         with get_list_of_videos(num_videos=3) as video_list:
             video_clips = VideoClips(video_list, 5, 5, num_workers=2)
-            self.assertEqual(video_clips.num_clips(), 1 + 2 + 3)
+            assert video_clips.num_clips() == 1 + 2 + 3
```

Comment:
technically we could have saved the `assert int_a == int_b` statements for other PRs but it was reasonably straightforward here so I went for it